src/block/block_write.c

```diff
@@ -220,7 +220,7 @@ __block_write_off(WT_SESSION_IMPL *session, WT_BLOCK *block, WT_ITEM *buf, wt_of
     WT_DECL_RET;
     WT_FH *fh;
     wt_off_t offset;
-    size_t align_size;
+    size_t align_size, unused_padding_size;
```

Comment:
```suggestion
    size_t align_size, padding_size;
```
The name `unused_padding_size` implies that there is "unused" padding, whereas this conveys the amount of padding from my reading.

Reply:
done, thanks.